src/components/CreateBeer.js

```diff
@@ -0,0 +1,110 @@
+/** @format */
+
+import axios from "axios";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default function CreateBeer() {
+	const [name, setName] = useState("");
+	const [tagline, setTagline] = useState("");
+	const [description, setDescription] = useState("");
+	const [first_brewed, setFirst_brewed] = useState("");
+	const [brewers_tips, setBrewers_tips] = useState("");
+	const [contributed_by, setContributed_by] = useState("");
+	const [attenuation_level, setAttenuation_level] = useState(0);
+
+	const navigate = useNavigate();
+
+	const createNewBeer = (e) => {
+		e.preventDefault();
```

Comment:
That way you add validation on the client side to avoid sending a request to the backend that will fail either way. So you have HTML validation + JS validation and later is good to also have the server side validation

And you can use the errorMessage to display an error message to the user so it can understand the problem :) 

```suggestion
		e.preventDefault();
		if (!name) {
		  setErrorMessage('We need the name of the beer')
		  return
		}
```